packages/ra-ui-materialui/src/input/TranslatableInputs.spec.tsx

```diff
@@ -126,17 +126,17 @@ describe('<TranslatableInputs />', () => {
         );
 
         fireEvent.change(
-            screen.getAllByLabelText('resources.undefined.fields.name')[0],
+            screen.getByLabelText('resources.undefined.fields.name.en'),
```

Comment:
Isn't that a BC? I'm worried the labels for TranslatableInputs will no longer be translated correctly, because the translation key has changed

Reply:
Indeed, good catch. I don't have a good solution yet for this.